.github/workflows/manual-pps.yml

```diff
@@ -0,0 +1,29 @@
+name: Manually Publish PPs
+on:
+  workflow_dispatch:
+
+jobs:
+  public-params:
+    name: Manually Generate & Upload PPs, hash-indexed
+    runs-on: ['memory:32GB']
```

Comment:
I think Groth16 parameter generation would require higher memory to be generated?